data/sql/updates/pending_db_world/rev_1728059831124992100.sql

```diff
@@ -0,0 +1,25 @@
+INSERT INTO creature_text
+(CreatureID, GroupID, ID, `Text`, `Type`, `Language`, Probability, Emote, Duration, Sound, BroadcastTextId, TextRange, comment)
```

Comment:
Before insert add always a delete, moreover add the backtick in all columns and in the table name

Reply:
Hey, I've implemented the changes mentioned here. Is it okay to group the deletes into one query like that or should it have been an individual delete for each?

Reply:
I think that what you did is ok and don't have other better options